.github/workflows/maven.yml

```diff
@@ -0,0 +1,211 @@
+name: Java CI with Maven and native builds
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  # Java version to use for the release
+  RELEASE_JAVA_VERSION: 11
+  UBUNTU_DIST: focal
```

Comment:
I appreciate the flexibility, but is there a good reason not to have this "hardcoded" in the Docerfile? The whole point of it is to have an old enough build so it works with ancient glibc...

Reply:
Yes, so anyone who wants to build this locally can override it.

Reply:
Do they need the Dockerfile to do that though?